current_test.go

```diff
@@ -46,39 +46,53 @@ func TestNewCurrent(t *testing.T) {
 	}
 }
 
+type current_weather struct {
```

Comment:
This should be camel case. Can you add a comment above it in the form of `// currentWeather ...` to keep the linter happy?